milestone.md

```diff
@@ -0,0 +1,199 @@
+
+- zdobycia liczby punktów
+
+    Ustalany jest próg punktowy. Jeżeli którakolwiek drużyna zdobędzie taką lub większą liczbę punktów, gra kończy się.
+
+- wyczerpanie zadań
+
+    Jeżeli istnieje drużyna, która nie ma dostępnych zadań bez zgłoszonej zatwierdzonej odpowiedzi, gra kończy się.
+    
+
+
+## Konstrukcja zadań
+Zadanie składa się z:
+    - tytułu
+    - opisu tekstowego
+    - listy subtasków, czyli typów załączników jakie użytkownik musi przesłać w jednej odpowiedzi
+    - liczby punktów za wykonanie zadania
+    - list prerequistes, czyli zadań, które muszą zostać wykonane, zanim to zadanie stanie się widoczne dla drużyny
+  
+# Frontend
+
+## Start - logowanie
+1. Użytkownik uruchamia aplikację
+2. Użytkownik wpisuje login i hasło
+3. Użytkownik klika loguj <br>
+4a. Komunikat (popup/toast) o błędzie w logowaniu <br>
+4b. Ekran z dwoma dużymi przyciskami do wyboru GM / gracz 
+5. Przejście do ekranu [lista gier](#lista-gier)
+
+## [GM] Lista gier
+1. Wyświetlają się gry o wszystkich statusach, w których użytkownik jest GMem. Sortowane wg statusu i czasu utworzenia.
+2. Można utworzyć nową grę przez kliknięcie przycisku (+), przejście do ekranu [nowa gra](#nowa-gra)
+3. Przejście do ekranu [edycja gry](#edycja-gry) po klinięciu na daną grę.
+
+## [GM] Nowa gra / edycja gry
+Ekran do edycji i tworzenia gry wyglądają identycznie. Przy tworzeniu nowej gry wszystkie pola są domyślnie puste, a przy edycji zostają automatycznie wypełnione obecnymi informacjami.
+1. Wprowadzana jest nazwa gry
+2. Wprowadzany jest opis gry <br>
+3a. Wybierane jest czy gra ma startować automatycznie o zadanje godzinie (checkbox) <br>
+3b. Pokazuje się pole, w którym wprowadzana jest data i godzina startu
+4. Wybierane jest czy gra ma mieć automatyczne zakończenie. Jeżeli to potrzebne, pojawiają się pola z miejscem na wpisanie czasu końca lub wymaganej liczby punktów.
+5. Wyświetla się lista zadań <br>
+6a. Symbolem "+" można dodać nowe zadanie <br>
+6b. Klikając na istniejące zadanie można je edytować
+7. Użytkownik klika na przycisk "zapisz" (lub z odpowienim symbolem), który przesyła wprowadzone informacje na serwer
+8. Użytkownik może kliknąć na przycisk "rozpocznij oczekiwanie na graczy" (albo lepsza nazwa), który prześle do serwera informacje o zmianie stanu gry i przejście na ekran [oczekiwania na graczy](). Przed wykonaniem akcji wyświetla się popup z prośbą o potwierdzenie.
+
+## [GM] Nowe zadanie
+1. Pola tekstowe na tytuł i treść zadania
+2. Pole na wpisanie liczby punktów, może być wprowadzone za pomocą klawiatury lub zmienione przy pomocy przycisków + / -.
+3. Pole, które umożliwia zaznaczenie checkboxami jakie zadania należy wykonać zanim to zadanie stanie się dostępne.
+4. Przycisk (+) / "Dodaj pole odpowiedzi", po jego kliknięciu z listy rozwijanej wybierany jest typ TEXT, PHOTO, QR lub AUDIO.
+5. Przycisk "Zapisz"
+
+## [GM] Oczekiwanie na gracz
+1. Wyświetla się tytuł gry oraz lista graczy, którzy zgłosili się do danej gry
```

Comment:
lista drużyn nie jest lepszym pomysłem?

Reply:
Lista drużyn, z rozbiciem na graczy. Dodatkowo GM mógłby zmienić drużynę gracza. Nie wiem tylko czy rozwiązać to przez drag&drop graczy między drużynami, czy może zwyczajne przyciski